test/attributes.js

```diff
@@ -62,15 +74,11 @@ describe('attributes', function() {
   });
   describe('Object.prototype property', function() {
     it('is not considered as a boolean attribute and shouldn\'t be omitted', function() {
```

Comment:
@staltz LGTM, even testing for single property is enough. Though it might be better to check using `true`/`false` instead of `'truthy'`/`null` for this single case to make it more obvious.